docs/Makefile

```diff
@@ -4,6 +4,9 @@ docs:
 	docker pull grafana/docs-base:latest
 	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'npm i && make webpack && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
 
+docs-local:
+	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'npm i && make webpack && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
```

Comment:
added this so you can run docs with locally built grafana/docs-base to see your recently added images

Reply:
I don't see any difference with this compared to `docs` target? Maybe I'm tired?

Reply:
The docs target downloads the latest from docker hub, or does it use local for you if local is newer?

Reply:
I tried again, and if I use `make docs` that do the docker pull before I loose all my newly added images that are not in the pushed docker image yet